src/protocol/subscriptions.cpp

```diff
@@ -31,73 +30,6 @@ namespace OpcUa
   }
 
   ////////////////////////////////////////////////////////
-  // CreateSubscriptionRequest
-  ////////////////////////////////////////////////////////
-
-  CreateSubscriptionRequest::CreateSubscriptionRequest()
-    : TypeId(CREATE_SUBSCRIPTION_REQUEST)
-  {
-  }
-
-  ////////////////////////////////////////////////////////
-  // DeleteSubscriptionRequest
-  ////////////////////////////////////////////////////////
-
-  DeleteSubscriptionRequest::DeleteSubscriptionRequest()
-    : TypeId(DELETE_SUBSCRIPTION_REQUEST)
-  {
-  }
-
-  DeleteSubscriptionResponse::DeleteSubscriptionResponse()
-    : TypeId(DELETE_SUBSCRIPTION_RESPONSE)
-  {
-  }
-
-
-  ////////////////////////////////////////////////////////
-  // SubscriptionData
-  ////////////////////////////////////////////////////////
-
-  SubscriptionData::SubscriptionData()
-    : RevisedPublishingInterval(100)
-    , RevisedLifetimeCount(30) //Should be 3 times keepalive
-    , RevizedMaxKeepAliveCount(10)
-  {
```

Comment:
These are important default values. I could not find them in the diff. Can you add a constructor for the autogenerated struct?